pkg/kubectl/cmd/describe.go

```diff
@@ -93,11 +94,12 @@ func NewCmdDescribe(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	kubectl.AddJsonFilenameFlag(cmd, &options.Filenames, usage)
 	cmdutil.AddRecursiveFlag(cmd, &options.Recursive)
 	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on")
+	cmd.Flags().BoolVar(&describerSettings.ShowEvents, "events", true, "If true, display also object events.")
```

Comment:
I prefer we call the flag "show-events=", we use that convention already with the `show-all` flag, and it parallels the name in the describer settings.

I would update the text to be "If true, display events related to the described object"